integration-tests/src/test/java/tachyon/security/MasterClientAuthenticationIntegrationTest.java

```diff
@@ -161,6 +161,8 @@ public void customAuthenticationDenyConnectTest() throws Exception {
             mLocalTachyonCluster.getMasterTachyonConf());
     Assert.assertFalse(masterClient.isConnected());
     masterClient.connect();
+    // should never reach here, but this silences the resource not closed warning
+    masterClient.close();
```

Comment:
I considered using @SuppressWarnings("resource") instead, but decided it was marginally better to close it instead of adding a suppression. I don't feel strongly about this at all - do you prefer suppressing?